src/packages/next/lib/with-customize.ts

```diff
@@ -73,6 +75,38 @@ export default async function withCustomize(
   customize.defaultComputeImage =
     softwareEnvs?.default ?? DEFAULT_COMPUTE_IMAGE;
 
+  customize.enabledPages = {
+    auth: {
+      try: !customize.account && customize.anonymousSignup,
+    },
+    about: {
+      index: customize.landingPages,
+      events: customize.isCommercial,
+      team: customize.landingPages,
+    },
+    compute: customize.isCommercial,
```

Comment:
hsy: maybe use the explicit enable or disable of compute servers for the site...